spec/routines/send_contact_info_confirmation_spec.rb

```diff
@@ -48,18 +48,11 @@
     end
 
     context 'confirmation_pin' do
-      it 'is blank when send_pin nil' do
```

Comment:
I found this to be an unreasonable expectation. I don't think it makes sense to have this as a requirement.